build/taskctl/contexts.yaml

```diff
@@ -15,7 +15,7 @@ contexts:
         - /app
         - --env-file
         - envfile
-        - amidostacks/runner-pwsh-data:0.4.18-stable
+        - amidostacks/runner-pwsh-data:0.4.19-unstable
```

Comment:
is there a reason for using an unstable version?

Reply:
the stacks data docker image is built from a feature branch so a unstable version was built. Here is the PR, https://github.com/Ensono/stacks-docker-images/pull/67. If you could approve then I will get that merged and build a stable version. Thanks